src/components/Comments/ContainerComment.js

```diff
@@ -16,29 +14,161 @@ import usePostContextHook, {CONTEXT_SOURCE} from '../../hooks/usePostContextHook
+            </ConnectorWrapper>
+          ) : null}
+        </React.Fragment>
+      ))}
+    </ContainerReply>
+  );
+};
+
+const ListComment = ({
+  indexFeed,
+  index,
+  onCommentLongPressed,
+  item,
+  isLast,
+  isLastInParent,
+  comments,
+  findCommentAndUpdate,
+  hideLeftConnector,
+  updateVote,
+  feedId,
+  onReplyButtonClick
+}) => {
+  return (
+    <TouchableWithoutFeedback key={index} onLongPress={() => onCommentLongPressed(item, 0)}>
+      <View>
+        <View key={`p${index}`}>
+          {item.user ? (
+            <Comment
+              feedId={feedId}
+              indexFeed={indexFeed}
+              key={`p${index}`}
+              comment={item}
+              user={item.user}
+              level={0}
+              time={item.created_at}
+              photo={item.user.data?.profile_pic_url}
+              isLast={isLast(index, item, comments)}
+              isLastInParent={isLastInParent(index, comments)}
+              showLeftConnector={false}
+              findCommentAndUpdate={findCommentAndUpdate}
+              onLongPress={onCommentLongPressed}
+              updateVote={updateVote}
+              onPress={(reactionId, replyUsername) =>
+                onReplyButtonClick(reactionId, replyUsername, 0)
+              }
+            />
+          ) : null}
+        </View>
+        {item?.children_counts?.comment > 0 && (
+          <ReplyComment
+            feedId={feedId}
+            hideLeftConnector={hideLeftConnector(index, item, comments)}
+            data={item.latest_children.comment}
+            countComment={item.children_counts.comment}
+            indexFeed={indexFeed}
+            findCommentAndUpdate={findCommentAndUpdate}
+            onCommentLongPressed={onCommentLongPressed}
+            updateVote={updateVote}
+            onReplyButtonClick={onReplyButtonClick}
+          />
+        )}
+      </View>
+    </TouchableWithoutFeedback>
+  );
+};
+
 const ContainerComment = ({
   feedId,
   comments,
   indexFeed,
   isLoading,
   refreshComment,
-  navigateToReplyView,
   findCommentAndUpdate,
   contextSource = CONTEXT_SOURCE.FEEDS,
   itemParent,
   updateVote,
-  isShortText
+  isShortText,
+  onReplyButtonClick
 }) => {
-  const navigation = useNavigation();
   const [, setSelectedCommentForDelete] = React.useState(null);
   const [selectedCommentLevelForDelete, setSelectedCommentLevelForDelete] = React.useState(0);
   const {isLast, isLastInParent, hideLeftConnector} = useContainerComment();
   const {calculationText, calculatedSizeScreen, calculatePaddingBtm} = usePostDetail();
   const {deleteCommentFromContext} = usePostContextHook(contextSource);
+  const onDeleteCommentClicked = async (item) => {
+    const response = await deleteComment(item?.id);
+    if (response?.success) {
+      deleteCommentFromContext(feedId, item?.id, selectedCommentLevelForDelete);
+      refreshComment();
+      SimpleToast.show('Comment has been deleted successfully');
+    }
+  };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of the `ListComment` component and its replacement with a new implementation directly in `ContainerComment` could potentially increase the complexity of this component. However, without seeing the full context of these changes, it's hard to say definitively whether this is an issue. If the `ListComment` component was only used here and its logic was simple, then this change might be justified.



<!-- This is an auto-generated comment by OSS CodeRabbit -->